domains/aryaejoumalai.json

```diff
@@ -0,0 +1,11 @@
+{
+    "description": "portfolio project",
+    "repo": "https://github.com/AryaEjoumalai/portfolio",
+    "owner": {
+        "username": "AryaEjoumalai",
+        "email": "aryaejoumalai@gmail.com",
+    },
+    "record": {
+        "CNAME": "aryaejoumalai.github.io"
```

Comment:
Your site has been flagged by Google. Can you double check your site is not phishing?